chore(middlewares/csp): set `useDefaults` to `true` by default

Fdawgs opened the pull request:
Closes #314

@EvanHahn not 100% on if this constitutes a major version bump?

EvanHahn commented on test/content-security-policy.test.ts:
```diff
@@ -372,9 +372,7 @@ describe("Content-Security-Policy middleware", () => {
-    }).toThrow(
-      /^Content-Security-Policy has no directives. Either set some or disable the header$/
-    );
+    }).not.toThrow();
```
I think this test should continue to `throw`, which means we'll need to set `useDefaults: false`.

Fdawgs replied:
Fixed in https://github.com/helmetjs/helmet/pull/316/commits/2c497fbb1dd47187852aee1dc51008d40958ade4

EvanHahn commented on middlewares/content-security-policy/README.md:
```diff
@@ -39,7 +39,7 @@ If no directives are supplied, the following policy is set (whitespace added for
     upgrade-insecure-requests
 
-You can use this default with the `useDefaults` option. `useDefaults` is `false` by default, but will be `true` in the next major version of this module.
+You can use this default with the `useDefaults` option. `useDefaults` is `true` by default.
```
Could you also make this change in the top-level `README.md`?

Fdawgs replied:
Amended in https://github.com/helmetjs/helmet/pull/316/commits/102234273c28ecdda6d27c8487214f2d1493411c

EvanHahn reviewed (changes requested):
Thanks for doing this!

1. Could we add at least one test that demonstrates overriding the defaults when `useDefaults` is completely unset? That test should fail on the old code and succeed in the new code.
1. Could you add the following to the top-level Helmet changelog:

   ```markdown
   ### Changed
   
   - `helmet.contentSecurityPolicy`: `useDefaults` option now defaults to `true`
   ```

Fdawgs commented on middlewares/content-security-policy/CHANGELOG.md:
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 4.0.0 - 2021-05-05
```
Amended in https://github.com/helmetjs/helmet/pull/316/commits/102234273c28ecdda6d27c8487214f2d1493411c

Fdawgs commented on middlewares/content-security-policy/package-overrides.json:
```diff
@@ -6,6 +6,6 @@
   ],
   "description": "Content Security Policy middleware",
-  "version": "3.4.0",
+  "version": "4.0.0",
```
Fixed in https://github.com/helmetjs/helmet/pull/316/commits/99dcf4c513ae71b5bb657db2505cd9ec4e1d0a68

Fdawgs commented:
> 
> 
> Thanks for doing this!
> 
>     1. Could we add at least one test that demonstrates overriding the defaults when `useDefaults` is completely unset? That test should fail on the old code and succeed in the new code.
> 
>     2. Could you add the following to the top-level Helmet changelog:
>        ```gfm
>        ### Changed
>        
>        - `helmet.contentSecurityPolicy`: `useDefaults` option now defaults to `true`
>        ```

Sure @EvanHahn, no problem!

1. Added in https://github.com/helmetjs/helmet/pull/316/commits/b32a28779c2fcb30f78e90abd1c0e6de6e7339c1
2. Added in https://github.com/helmetjs/helmet/pull/316/commits/102234273c28ecdda6d27c8487214f2d1493411c

EvanHahn commented:
Thanks for doing this! I'll take another look soon.

EvanHahn commented:
I haven't forgotten about this, but it's low on my list because I don't intend to release Helmet v5 soon. Still hoping to review this in the next couple of weeks, though!

EvanHahn reviewed (approved):
Thanks!

EvanHahn commented:
I made a mistake when rebasing the `v5.x` branch, so I "redid" this commit in 16f883fdf50ab473f746f0032f3ddb3caa44b85b.
